indico/modules/events/editing/controllers/frontend.py

```diff
@@ -51,10 +51,19 @@ def _process(self):
 
 
 class RHManageEditingFileTypes(RHEditingManagementBase):
+    def _process_args(self):
+        RHEditingManagementBase._process_args(self)
+        self.editable_type = request.view_args['type']
```

Comment:
Should we allow accessing/managing an editable type that is not enabled?

Reply:
I'm leaning towards yes, since there's no harm in it - I'd just check it for user/editor-facing endpoints (#4375).

Just like we also allow accessing `/manage/contributions` for a meeting, even though this page is usually not used/shown outside conferences.

Reply:
I agree but we should also display it in the editable list even if it is disabled, as the only way to edit the disabled ones would be via typing the url for them.

Reply:
That's a very good point! Maybe showing it but keeping it grayed-out to make it clear it's disabled?

Reply:
Yeah, could go in a separate commit/task though

Reply:
Yep, I was about to suggest that you could include it in your current task but I think it makes even more sense to have a separate task for it.

Reply:
Actually, thinking about it again, do we need this? If you enable a type but don't open submission/editing for it nor assign anyone, the type would not show up for users/editors anyway, and thus there's no real need to configure it while it's disabled.

Reply:
I thought that one might want to configure them all first before enabling them.